### ISSUE TITLE
GDPR official module translation issue

### ISSUE THREAD

**Issue by gringo1er** [PrestaShop/PrestaShop#26352]
#### Describe the bug

Hello. Our website is in french. So we have translated most part of the website (modules, contents, template) in french. 
We translated GDPR texts and buttons for our french customer, through international / translation back office access.
Unfortunately, the personal date management info, inside the customers accounts, remains in english.

#### Expected behavior

French translated texts in customers accounts

#### Steps to Reproduce

Translate text in french, and go to a customer account.

**Screenshots**
![gdpr1](https://user-images.githubusercontent.com/9809622/138254061-2dd8d9f4-56f3-44d7-bcc5-09a974716032.jpg)
![gdpr2](https://user-images.githubusercontent.com/9809622/138254071-5861041b-6a51-4998-b0df-38bba3f0d5ce.jpg)

#### Additional information

* PrestaShop version: 1.7.7.5, 1.7.7.8, 1.7.8.0
* PHP version: 7.3, 7.4 (PS 1.7.8.0)

**Comment by hibatallahAouadni** [comment]
Hello @gringo1er 

About the module traduction, please follow these steps to change the translations:

1. **BO**>**IMPROVE**>**International**>**Translations**
2. Select **Installed modules translations** as **Type of translation**
3. Select the module
4. Select your language
5. Click on **Modify**
6. Enjoy it :wink: 

**BO:**
![image](https://user-images.githubusercontent.com/23308427/138272109-56f72f98-3f44-4ef6-9ec5-155b9280eb8e.png)

**FO:**
![image](https://user-images.githubusercontent.com/23308427/138272144-ca4c6be5-2ecb-4152-8d6b-447928f206ab.png)

Closing this issue as it's not a bug.
PS: if you have any other questions, you can always ask here even when the issue is closed, I will reply as soon as possible.

Thanks!

**Comment by gringo1er** [comment]
@hibatallahAouadni 
Hi. It's exactly what we did.... 
![gdpr3](https://user-images.githubusercontent.com/9809622/138281198-4392297c-47ac-4552-ae2a-8bdc645e1297.jpg)

That's what I explained... Except that my back office doesn't look like your back office
![gdpr1](https://user-images.githubusercontent.com/9809622/138281511-15533538-249c-4f2b-9493-0843ea5058a9.jpg)

And it doesn't work.

We have tanslated many modules, it works except this one.
